src/convertor/OndcConvertor.js

```diff
@@ -0,0 +1,38 @@
+import PlatformFormatter from '../formatter/PlatformFormatter';
+
+export default class OndcConvertor {
+  static async convert(ondcMappedTags) {
+    const convertedOndcResponse = await Promise.all(
+      ondcMappedTags.map(async (tag) => {
+        const ondcValueType = ((tag.ondcDataType === 'boolean' || tag.platformValue === '')
+          ? '' : `| to${tag.ondcDataType}`);
+        const platformResponseValue = (tag.ondcDataType === 'boolean'
+          ? tag.platformValue : `"${tag.platformValue.toString().replace(/"/g, '\\"')}"`);
+        const ondcTagKeys = tag.ondc.split('.');
+        const ondcItemValue = `${platformResponseValue} ${ondcValueType}`;
+        const ondcFilter = `. | if ( ${ondcTagKeys.length} > 1) 
```

Comment:
There is lot of logic here - write a smaller unit which is one tag to its object

Reply:
updated